src/Goerli-DssSpell.sol

```diff
@@ -86,278 +27,107 @@ contract DssSpellAction is DssAction {
+            pip:                   PIP_GUNI,
+            isLiquidatable:        false,
+            isOSM:                 true,
+            whitelistOSM:          false,
+            ilkDebtCeiling:        3 * MILLION,
+            minVaultAmount:        10 * THOUSAND,
+            maxLiquidationAmount:  5 * MILLION,
+            liquidationPenalty:    1300,
+            ilkStabilityFee:       ONE_PCT_RATE,
+            startingPriceFactor:   10500,
+            breakerTolerance:      9500, // Allows for a 5% hourly price drop before disabling liquidations
+            auctionDuration:       220 minutes,
+            permittedDrop:         9000,
+            liquidationRatio:      10500,
+            kprFlatReward:         300,
+            kprPctReward:          10 // 0.1%
+        });
 
-        // Rely Oracles team address in all the medians except MATIC that was created by them
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_ETH")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAT")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_WBTC")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_ZRX")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_KNC")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_MANA")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_USDT")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_COMP")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_LRC")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_LINK")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAL")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_YFI")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_UNI")).src(),  0x1f42e41A34B71606FcC60b4e624243b365D99745);
-        DssExecLib.authorize(OsmAbstract(DssExecLib.getChangelogAddress("PIP_AAVE")).src(), 0x1f42e41A34B71606FcC60b4e624243b365D99745);
+        DssExecLib.addNewCollateral(GUNI_A);
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC1-A", 10 * MILLION, 10 * MILLION, 8 hours);
```

Comment:
There is something wrong here, 10M debt ceiling and 10M gap doesn't really make much sense.
I guess it was wanted to set `gap`to 3M like the initial debt ceiling.

Reply:
This is intended behaviour.

Reply:
Ok if that's the case, please set the ilk debt ceiling to 10M directly as anyway as soon as this is up anyway can bump it to that value.

Reply:
done